### DISCUSSION TITLE
Event Login for participants

### DISCUSSION THREAD

**Discussion by markusthiel** [HiEventsDev/Hi.Events#52] [Ideas]
What would be really great: if the frontend for participants had a nice landing page with a login function to have their own area where they can, for example, start surveys, receive pop-up notifications, access FAQs, and above all, share updates and changes about the event. There id an extensive App named lineupr.

**Comment by daveearley** [comment]
That's a nice idea alright. In the near future I'll be adding the ability to add FAQs and event schedules etc. The plan was to only have them publicly available on the event page, but I like the idea of having the option to only expose them to attendees.

**Reply by justynpride** [reply]
A related but slightly different spin would be for the public user once logged in to view their order and even past orders. I’ve run some large scale event where was permitted users to edit their live booking until a set date. This allowed for switching of tickets, adding new tickets, and changing names. At the end of the process the system notified of any outstanding fees and allowed for additional payments to be taken, or refund request to be made. 
I can definitely see the potential for a user system for those who have made that orders.

**Reply by jenton93** [reply]
So I have been on the hunt for a ticketing platform. Where someone books x number of tickets. Can firstly increase this booking keeping it all in 1 place and secondly the ability to buy upgrades to the booking. That can be per head or per item and also can be forced to be everyone in the group. Like free drinks only available to the whole group. But this could be purchased at a later date to the main ticket.
